.vscode/settings.json

```diff
@@ -0,0 +1,12 @@
+{
+  "[javascript]": {
+      "editor.defaultFormatter": "dbaeumer.vscode-eslint",
+      "editor.codeActionsOnSave": {
+          "source.fixAll": "explicit"
+      }
+  },
+
+  "workbench.colorCustomizations": {
+      "statusBar.background": "#178bb9"
+  }
+}
```

Comment:
I'm 👎 on this, see eg. https://github.com/microsoft/vscode/issues/206802

I find it quite jarring when my settings are changed due to an `.vscode/settings.json` – and unless all contributors are forced to use VSCode it feels odd to enforce this just on VSCode. Use eg. git hooks on `push` to avoid having people push unformatted code.

Reply:
A better way is to make it a devcontainer and add this as part of its vscode customization section: https://containers.dev/supporting#visual-studio-code

One can make this a devcontainer template that Fastify repositories can easily set up: https://containers.dev/implementors/templates/

Reply:
The original guide detailed setting up a _Fastify specific_ VSCode instance. Again, I would revert this PR to commit [a068946 (#5890)](https://github.com/fastify/fastify/pull/5890/commits/a068946d8894034f9a63f5d76c2d253c44046dda) and then update the content as necessary.